packages/dht/src/dht/DhtNode.ts

```diff
@@ -157,6 +157,11 @@ export class DhtNode extends EventEmitter<Events> implements ITransport {
                 throw new Error(`Invalid nodeId, the length of the nodeId should be ${expectedNodeIdLength}`)
             }
         }
+        if (this.config.peerDescriptor !== undefined) {
+            if (this.config.peerDescriptor.nodeId.length !== KADEMLIA_ID_LENGTH_IN_BYTES) {
```

Comment:
peerDescriptor.nodeId should be `KADEMLIA_ID_LENGTH_IN_BYTES` but above in the code this.config.nodeId should be `KADEMLIA_ID_LENGTH_IN_BYTES * 2`. Not sure I follow, should these be the same?

Reply:
The type is different. If a nodeId is given explicitly is given in string as hex. In the PeerDescriptor it is a Uint8Array.

Reply:
Got ya